.changeset/three-llamas-know.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lg-charts/core': minor
```

Comment:
Should this be `major`? From the ticket, it looks like no one would even be using it, so this is probably ok